src/WinGetUtil/WinGetUtil.h

```diff
@@ -76,9 +76,11 @@ extern "C"
     // Validates a given manifest. Returns a bool for validation result and
     // a string representing validation errors if validation failed.
     WINGET_UTIL_API WinGetValidateManifest(

```

Comment:
>WinGetValidateManifest [](start = 20, length = 22)

This will need to be a new function and the old one will need to continue to work with it's existing signature.